arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,410 @@
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        },
+                        "description": "The required parameters for createing or updating a Azure SQL Server auditing policy."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "AuditingPolicy"
+                ],
+                "operationId": "AuditingPolicy_GetServerPolicy",
+                "description": "Returns an Azure SQL Server auditing policy.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "DatabaseAuditingPolicyProperties": {
+            "properties": {
+                "useServerDefault": {
+                    "type": "string",
+                    "description": "Indicates whether the default server auditing policy is used for that Azure SQL Database."
+                },
+                "auditingState": {
+                    "type": "string",
+                    "description": "Gets the auditing state of the Azure SQL Server auditing policy."
+                },
+                "eventTypesToAudit": {
+                    "type": "string",
+                    "description": "Gets the events types of the Azure SQL Server auditing policy."
```

Comment:
@yoavrubin  Could you please comment on this.

---
In reply to: [80096363](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80096363) [](ancestors = 80096363)